tests/testthat/test-get_latest_resource_id.R

```diff
@@ -0,0 +1,7 @@
+test_that("returns data for a dataset that is listed", {
+  expect_no_error(get_latest_resource_id("gp-practice-populations"))
+})
+
+test_that("returns error for a dataset that is not listed", {
+  expect_error(get_latest_resource_id("hospital-codes"))
```

Comment:
It's good to add some regex / text to check for the exact expected error (as opposed to an error, but not the one you expected!)
```r
  expect_error(get_latest_resource_id("hospital-codes"), "<expected error regex>")
```